app.js

```diff
@@ -81,28 +90,60 @@ var imageConstructorArray = [
   shark,
   sweep,
   tauntaun,
-  thumbs,
   unicorn,
   usb,
   waterCan,
   wineGlass,
-] ;
+];
+var percentArray = [];
+var threePic = [];
+var oldPics = [];
+var numOfPics = 3;
 // populates three images pulled at an index from the array. the index is a random number
 function populateThree(){
-  for (var i = 0; i < 3; i++) {
+  oldPics = threePic;
```

Comment:
Can you think of better names for these variables? Neither of these are particularly clear. For example, these arrays both hold numbers, not pictures.